doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -188,6 +188,21 @@ Field        | Type       | Meaning
 `export_id ` | [`text`]   | The ID of the index or materialized view that created the dataflow. Corresponds to [`mz_compute_exports.export_id`](#mz_compute_exports).
 `time`       | [`mz_timestamp`] | The next timestamp at which the output may change.
 
+### `mz_cluster_replica_frontiers`
+
+The `mz_cluster_replica_frontiers` table describes the frontiers of each [dataflow] in the system.
+[`mz_compute_frontiers`](#mz_compute_frontiers) is similar to this table, but `mz_compute_frontiers`
+exposes the frontiers known to the compute replicas while `mz_cluster_replica_frontiers` contains
+the frontiers the coordinator is aware of.
+
+At this time, we do not make any guarantees about the exactness or freshness of these numbers.
```

Comment:
I think you can delete this -- it seems copied from `mz_cluster_replica_metrics`, but I included that note for that table specifically because the Kubernetes `metrics-server` doesn't guarantee the accuracy of the numbers.

Reply:
It is copied :-) I included it because of the delay, ie. we update the table only periodically. But you are right about the exactnes, that is not an issue in this case. Will remove that